enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/core/replication/ReplicatedContent.java

```diff
@@ -24,4 +24,13 @@
  */
 public interface ReplicatedContent
 {
+    default boolean hasSize()
+    {
+        return false;
+    }
+
+    default long size()
+    {
+        throw new UnsupportedOperationException();
+    }
```

Comment:
Yea the naming can probably be improved. I guess everything has a payload though.